source/renderer/app/components/staking/rewards/StakingRewardsForIncentivizedTestnet.js

```diff
@@ -0,0 +1,258 @@
+  },
+  note: {
+    id: 'staking.rewards.note',
+    defaultMessage:
+      '!!!Rewards from stake delegation are automatically collected into your reward account.',
+    description: 'Rewards description text on staking rewards page',
+  },
+});
+
+type Props = {
+  rewards: Array<RewardForIncentivizedTestnet>,
+  isLoading: boolean,
+  isExporting: boolean,
+  onLearnMoreClick: Function,
+  onExportCsv: Function,
+};
+
+type State = {
+  rewardsOrder: string,
+  rewardsSortBy: string,
+};
+
+@observer
+export default class StakingRewardsForIncentivizedTestnet extends Component<
+  Props,
+  State
+> {
+  static contextTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  static defaultProps = {
+    isLoading: false,
+    isExporting: false,
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      rewardsOrder: 'desc',
+      rewardsSortBy: 'wallet',
+    };
+  }
+
+  handleExportCsv = (
+    availableTableHeaders: Array<any>,
+    sortedRewards: Array<RewardForIncentivizedTestnet>
+  ) => {
+    const { onExportCsv } = this.props;
+    const exportedHeader = availableTableHeaders.map(header => header.title);
+    const exportedBody = sortedRewards.map(reward => {
+      const rewardWallet = get(reward, 'wallet', '');
+      const rewardAmount = get(reward, 'reward', 0);
+
+      return [rewardWallet, `${rewardAmount} ADA`];
+    });
+    const exportedContent = [exportedHeader, ...exportedBody];
+
+    onExportCsv(exportedContent);
+  };
+
+  render() {
+    const { rewards, isLoading, isExporting, onLearnMoreClick } = this.props;
+    const { rewardsOrder, rewardsSortBy } = this.state;
+    const { intl } = this.context;
+    const noRewards = !isLoading && ((rewards && !rewards.length) || !rewards);
+    const showRewards = rewards && rewards.length > 0 && !isLoading;
+    const sortedRewards = showRewards
+      ? orderBy(rewards, rewardsSortBy, rewardsOrder)
+      : [];
+    const availableTableHeaders = [
+      {
+        name: 'wallet',
+        title: intl.formatMessage(messages.tableHeaderWallet),
+      },
+      {
+        name: 'reward',
+        title: intl.formatMessage(messages.tableHeaderReward),
+      },
+    ];
+    const exportCsvButtonLabel = isExporting ? (
+      <div className={styles.exportingSpinnerWrapper}>
+        <LoadingSpinner />
+      </div>
+    ) : (
+      <>
+        <div className={styles.actionLabel}>
+          {intl.formatMessage(messages.exportButtonLabel)}
+        </div>
+        <SVGInline svg={downloadIcon} className={styles.downloadIcon} />
+      </>
+    );
+
+    return (
+      <StakingPageScrollContext.Consumer>
+        {context => (
+          <div className={styles.component}>
+            <div className={styles.headerWrapper}>
+              <div className={styles.title}>
+                {intl.formatMessage(messages.title)}
+              </div>
+              {!noRewards && (
+                <Button
+                  className={classNames([
```

Comment:
Extract that to ButtonClasses variable as you have for `sortIconClasses `

Reply:
This is a bit odd, because it uses context, so I had to define a new method instead of variable.
Anyway, did that to prevent back and forth